mlir/lib/TableGen/Pattern.cpp

```diff
@@ -503,6 +552,13 @@ bool SymbolInfoMap::bindAttr(StringRef symbol) {
   return symbolInfoMap.count(inserted->first) == 1;
 }
 
+bool SymbolInfoMap::bindProp(StringRef symbol,
+                             const PropConstraint &constraint) {
+  auto inserted =
```

Comment:
Not due to you, but this looks weird, not sure why its done this way. It inserts but then counts if inserted (its an unordered_map it seems, so don't think count can be > 1), why not just return inserted->second.

Reply:
It's an unordered *multi*map - so you can specifically both check for two results and - I suspect - determine what there's a conflict with